### DISCUSSION TITLE
Welcome to terraform-provider-libvirt community discussions!

### DISCUSSION THREAD

**Discussion by dmacvicar** [dmacvicar/terraform-provider-libvirt#1099] [Announcements]
## 👋 Welcome!
  I have enabled Discussions as a place for the community to connect with each other. We hope that you:
  * Ask questions you’re wondering about.
  * Help other community members with problems.
  * Occasionally, we will look at common problems and turn them into fixes and better user experience

**Comment by jeje232** [comment]
Hello,

I used v0.7.6 and define a ssh ed25519 key in the local uri like that:
qemu+ssh://root@172.16.0.154/system?socket=/var/run/libvirt/libvirt-sock&sshauth=privkey&keyfile=/home/terraform/.ssh/id_ed25519

I seems it's didn't work now with the 0.8.0 version. I did not find syntax to use .ssh keys to enable the provider to access to the libvirt server.

Thank you very much

**Reply by dmacvicar** [reply]
https://github.com/dmacvicar/terraform-provider-libvirt/discussions/categories/q-a
